calcom/cal.com
# fix: check for single email in TextAreaField

## What does this PR do?

Fixes #10146

https://www.loom.com/share/71752c0d84f2443593987fbcdd580786?sid=f38e0015-67d6-4ded-9f15-8bc9df1c5bbb

## Type of change




## How should this be tested?

1. Goto /settings/organizations/new
2. Fill in necessary details until `onboard-admins` page
3. Enter 1 email and check if the toast notification says `{{email}} has been invited`
4. Enter multiple emails and check if the toast notification says `{{userCount}} users have been invited`

## Mandatory Tasks
